build.cmd

```diff
@@ -2,11 +2,11 @@
 
 echo %* | findstr /C:"-pack" >nul
 if %errorlevel%==0 (
-    set PackInstaller=
+    set SkipBuildingInstallers=
 ) else (
     REM disable crossgen for inner-loop builds to save a ton of time
-    set PackInstaller=/p:PackInstaller=false
+    set SkipBuildingInstallers=/p:SkipBuildingInstallers=false
```

Comment:
This changes the default, right?

Reply:
No, the default should be the same as before.

Reply:
OK, maybe I'm missing the full picture but `PackInstaller` and `SkipBuildingInstallers` seem to mean the opposite intent, but the value is `false` in both cases.

Reply:
Oh of course, thanks for pointing that out. Will fix.